### PR TITLE
Remove shadow map variant (change directional shadow + punctual to single)

### PR DESCRIPTION
**Pull request by sebastienlagarde**
### Purpose of this PR
This PR remove punctual and directional variant in favor of a single settings:
LOW, MEDIUM and HIG shadow quality.

Directional are move from PCSS to Tent7x7 as PCSS don't work correctly with directional

---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
### Testing status
**Katana Tests**: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=hdrp-less-shadow-variant&automation-tools_branch=add-platform-filter&unity_branch=trunk#

**Manual Tests**: Manually with our test scene

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.cs [hunk 1]
```diff
@@ -608,18 +608,14 @@ static void DrawBakedShadowsContent(SerializedHDLight serialized, Editor owner)
         
         static bool IsShadowSettings(HDShadowQuality quality, SerializedHDLight serialized, Editor owner)
```
Comment (EvgeniiG) [inline com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.cs]: IsShadowSettings ?? Returns true if it's shadow settings?

File: com.unity.render-pipelines.high-definition/Runtime/Sky/SkyRenderingContext.cs [hunk 2]
```diff
@@ -280,7 +280,7 @@ public void RenderSky(SkyUpdateContext skyContext, HDCamera hdCamera, Light sunL
 #if UNITY_2019_1_OR_NEWER
                     m_BuiltinParameters.pixelCoordToViewDirMatrix = HDUtils.ComputePixelCoordToWorldSpaceViewDirectionMatrix(hdCamera.camera.GetGateFittedFieldOfView() * Mathf.Deg2Rad, hdCamera.camera.GetGateFittedLensShift(), hdCamera.screenSize, hdCamera.viewMatrix, false);
 #else
-                    m_BuiltinParameters.pixelCoordToViewDirMatrix = HDUtils.ComputePixelCoordToWorldSpaceViewDirectionMatrix(hdCamera.camera.fieldOfView * Mathf.Deg2Rad, hdCamera.camera.GetGateFittedLensShift(), hdCamera.screenSize, hdCamera.viewMatrix, false);
+                    m_BuiltinParameters.pixelCoordToViewDirMatrix = HDUtils.ComputePixelCoordToWorldSpaceViewDirectionMatrix(hdCamera.camera.fieldOfView * Mathf.Deg2Rad, Vector2.zero, hdCamera.screenSize, hdCamera.viewMatrix, false);
```
Comment (EvgeniiG) [inline com.unity.render-pipelines.high-definition/Runtime/Sky/SkyRenderingContext.cs]: Why was this changed?

Comment (EvgeniiG) [reply com.unity.render-pipelines.high-definition/Runtime/Sky/SkyRenderingContext.cs]: To be more consistent with the sky call above?

Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Runtime/Sky/SkyRenderingContext.cs]: My mistake, it is a backport of commit to compil on 2018.3. should ahve been in HDRP/Staging

File: com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.cs (hunk 1 above)
Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.cs]: Not part of this PR change.
But agree, name of this function is weird, it should have been haveShadowSettingsUI

File: com.unity.render-pipelines.high-definition/Runtime/Sky/SkyRenderingContext.cs (hunk 2 above)
Comment (JulienIgnace-Unity) [inline com.unity.render-pipelines.high-definition/Runtime/Sky/SkyRenderingContext.cs]: Is this intended in this PR?
